resources/system-overview.md

```diff
@@ -0,0 +1,105 @@
+
+### Data Sources
+Most sources will be required for both application and renewal. Often data sources will specify a real time API for applications to facilitate low-latency responses to user interaction on a portal, and a batch interface to allow for renewal operations to avoid overloading the real-time API.
+
+The federal Data Hub provides proxy interface to a number of data sources:
+-	Social Security Administration for SSN validation, SSDI, and deaths
+-	Internal Revenue Service for income and household composition
+-	Department of Labor for unemployment insurance income
+-	The Work Number for income
+-	Department of Homeland Security for immigration status
+-	Interstate exchange for cross-enrollment
+-	National Change of Address 
+
+Some states maintain separate interfaces to these services instead of using the Hub.
+
+These services, including those proxied via the Hub do not cache results, and fee-based services may bill for each query. Client-side caching and rate limiting are therefore very important. Data source interfaces should be configured to never make the same query within a source-specific lock out time after a successful response has been received. However, it is rare to observe these limitations or caches in real systems.
+
+Some sources may need to be polled periodically to detect Life Status Change events such as moving across state lines, getting or losing a job. These events can trigger notifications requesting more information.
+
+Horizontally-integrated systems will need to determine the appropriate schedule for each data source based on beneficiary membership. Data collected for one benefit program may trigger life change events for other programs.
+
+### Rules Engine
+This is where program eligibility policy belongs. The name “rules engine” isn’t intended to imply a specific implementation, it is a functional definition of the module that evaluates the rules and determines eligibility. The rules engine never disenrolls an individual. During application it returns “Eligible” or “Ineligible”. During renewal it returns “Ex Parte Renewal” or “Manual Renewal”. The set of rules that are evaluated at renewal time is a subset of rules that are run at application.
```

Comment:
It might be worth adding here that some states have additional end states; "Pending Eligibility", for example, or "Conditional". These are generally mapped into "eligible" or "ineligible" afterwards, but not always!

Reply:
done! although pulling on this thread maybe means there should be an expanded section that includes discussion of R&O style pauses?